tests/expectations/metrics/test_core.py

```diff
@@ -34,7 +39,7 @@
     build_sa_engine,
     build_spark_engine,
 )
-from great_expectations.util import isclose
+from great_expectations.util import get_context, isclose
```

Comment:
@TrangPham The context import is indicative of a potential integration style test (this module is for unit tests).  Thank you.

Reply:
I moved the test to test_sparkdf_execution_engine